app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components
+@import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType
+@import uk.gov.hmrc.merchandiseinbaggage.views.html.Layout
+@import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig
+@import uk.gov.hmrc.merchandiseinbaggage.controllers.routes._
+
+@this(
+        layout: Layout,
+        h1: components.h1,
+        p: components.p,
+        link: components.link
+)
+
+@(declarationType: DeclarationType)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@*TODO Back and retry links need correcting *@
+
+@layout(pageTitle = Some(messages("declarationNotFound.title")), maybeBackButtonUrl = Some(ExciseAndRestrictedGoodsController.onSubmit), maybeDeclarationType = None) {
+    @h1(("declarationNotFound.heading"))
+    @p(Html(messages("declarationNotFound.li1")))
+    @p(Html(messages("declarationNotFound.li2")))
+    @p(Html(messages("declarationNotFound.li3")))
+    <p class="govuk-body">
+        @messages("declarationNotFound.li4.a.text")
+        @link("TODO /retrieve-declaration", "declarationNotFound.li4.b.text")
+        @messages("declarationNotFound.li4.c.text")
+        @link( ExciseAndRestrictedGoodsController.onSubmit.url, "declarationNotFound.li4.d.text")
```

Comment:
if its for a new journey, this link should point to /goods-destination or something

Reply:
This is what the ticket and prototype require /excise-and-restricted-goods

Reply:
I think Suresh is right, first page in the journey is /goods-destination -> our prototype is becoming a mess

regardless of that, this link is going to the POST endpoint, you need to go to the .onPageLoad() method

Reply:
change to /goods-destination